src/quo2/components/avatars/user_avatar.cljs

```diff
@@ -82,14 +82,13 @@
    :border-radius inner-dimensions})
 
 (defn container [inner-dimensions outer-dimensions & children]
-  (let [dark-kw (if dark? :dark :light)]
-    [rn/view {:style (merge {:background-color (colors/custom-color
```

Comment:
Why changes in user avatar?

Reply:
I added this fix because when the team updated the colors ns, The color for the component was different.

Reply:
Apparently this issue isn't happening on latest dev
Reverted